lib/filterx/object-string.c

```diff
@@ -187,6 +187,7 @@ _string_freeze(FilterXObject **pself)
       *pself = frozen_string;
       return;
     }
+  self->hash = _filterx_string_hash(self);
```

Comment:
`_filterx_string_hash()` already has the side-effect to store its value in `self->hash`.

Reply:
Will fix that too.